src/main/java/liquibase/ext/databricks/sqlgenerator/CreateTableGeneratorDatabricks.java

```diff
@@ -1,22 +1,43 @@
 package liquibase.ext.databricks.sqlgenerator;
 
 
+import liquibase.database.Database;
 import liquibase.exception.ValidationErrors;
 import liquibase.ext.databricks.change.createTable.CreateTableStatementDatabricks;
 import liquibase.ext.databricks.database.DatabricksDatabase;
-import liquibase.sqlgenerator.core.CreateTableGenerator;
-import liquibase.database.Database;
 import liquibase.sql.Sql;
 import liquibase.sql.UnparsedSql;
 import liquibase.sqlgenerator.SqlGeneratorChain;
+import liquibase.sqlgenerator.core.CreateTableGenerator;
 import liquibase.statement.core.CreateTableStatement;
-import liquibase.structure.DatabaseObject;
 import org.apache.commons.lang3.StringUtils;
 
-import java.util.List;
+import java.util.*;
+
+import static java.util.stream.Collectors.joining;
 
 public class CreateTableGeneratorDatabricks extends CreateTableGenerator {
 
+    private static final Set<String> ESSENTIAL_PROPERTIES = Set.of(
+            "'delta.columnMapping.mode'",
+            "'delta.enableDeletionVectors'",
+            "'delta.feature.allowColumnDefaults'",
+            "'delta.logRetentionDuration'",
+            "'delta.deletedFileRetentionDuration'",
+            "'delta.targetFileSize'",
+            "'delta.enableChangeDataFeed'"
+    );
+
+    private static final Map<String, String> DEFAULT_VALUES = Map.of(
+            "'delta.columnMapping.mode'", "'name'",
+            "'delta.enableDeletionVectors'", "true",
+            "'delta.feature.allowColumnDefaults'", "'supported'",
+            "'delta.logRetentionDuration'", "'30 days'",
+            "'delta.deletedFileRetentionDuration'", "'7 days'",
+            "'delta.targetFileSize'", "134217728",
```

Comment:
Where did those values came from? Do we need them?
Cause before we only had 3 columns with default values: 
* 'delta.feature.allowColumnDefaults' = 'supported'
* 'delta.columnMapping.mode' = 'name'
* 'delta.enableDeletionVectors' = true